packages/address-mocks/src/fixtures/countries_en.ts

```diff
@@ -1,11437 +1,4 @@
-const data = {
```

Comment:
To keep the nicer formating and make this easier to read and update in the future we could do:

```
//prettier-ignore
const data = JSON.parse(`{
  data: BLAH
}`)